var/spack/repos/builtin/packages/gptune/package.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Gptune(CMakePackage):
+    """GPTune is an autotuning framework that relies on multitask and transfer
+    learnings to help solve the underlying black-box optimization problem using
+    Bayesian optimization methodologies."""
+
+    homepage = "https://gptune.lbl.gov/"
+    git      = "https://github.com/gptune/GPTune.git"
+    maintainers = ['liuyangzhuan']
+
+    version('master', branch='master')
+
+    variant('app', default=False, description='Build all HPC application examples')
+
+    depends_on('mpi', type=('build', 'link', 'run'))
+    depends_on('cmake@3.3:', type='build')
+    depends_on('jq', type='run')
+    depends_on('blas', type=('build', 'run'))
+    depends_on('lapack', type=('build', 'run'))
+    depends_on('scalapack', type=('build', 'run'))
```

Comment:
These should be `type='link'`. You're explicitly linking to them in `cmake_args` below. I don't think anything even gets installed to `prefix.bin`, so build/run wouldn't do anything.

Reply:
ok done.